tests/test_entry.py

```diff
@@ -19,12 +21,17 @@ def test_basic_save(self):
         e = self.create_entry()
         e.save()
 
+        if celery_main_version < 4:
+            kwargs_default = None
```

Comment:
would it make sense to hoist this assignment up into the import section, as CELERY_CONFIG_DEFAULT_KWARGS, which would allow removal of this conditional?

Reply:
Good idea, I will make the change.